lib/resources/pip.rb

```diff
@@ -62,18 +63,52 @@ def to_s
 
     private
 
+    def cmd
+      @__cmd ||= inspec.command("#{@pip_cmd} show #{@package_name}")
+    end
+
+    def cmd_successful?
+      result = inspec.command("#{@pip_cmd} show #{@package_name}")
```

Comment:
Could we just use `cmd` here instead of doing the `inspec.command` again?

Reply:
Fixed